index.js

```diff
@@ -51,10 +51,41 @@ var exec = require('child_process').exec
 var fs = require('fs')
 var accessSync = require('fs-access').sync
 var pkgPath = path.resolve(process.cwd(), './package.json')
+var cordovaPath = path.resolve(process.cwd(), './config.xml')
 var pkg = require(pkgPath)
 var semver = require('semver')
 var util = require('util')
 
+function isCordovaProject () {
+  return !!fs.existsSync(cordovaPath)
```

Comment:
I think we should make this slightly stricter; check whether file exists, and also confirm that it contains `android-versionCode` perhaps? I can imagine projects having a config.xml that are not cordova.